src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,65 @@
+package parser;
+
+import org.testng.annotations.*;
+import org.testng.asserts.SoftAssert;
+import shop.Cart;
+
+import java.io.*;
+
+import static org.testng.Assert.*;
+
+
+class JsonParserTest {
+
+    private JsonParser parser;
+    public static final String CART_NAME = "andrew-cart";
+    @DataProvider(name = "badFilePaths")
+    public Object[][]createData1() {
+        return new Object[][] {
+                { "badfileone"},
+                { "badfiletwo"},
+                { "badfilethree"},
+                { "badfilefour"},
+                { "badfilefive"},
+        };
+    }
+
+
+    @BeforeTest
```

Comment:
it is better to create test data before each method. Use '@BeforeMethod' annotation